pkgs/development/tools/flamegraph/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, perl }:
+
+stdenv.mkDerivation {
+  name = "FlameGraph";
```

Comment:
`name` should contain the version. If there are no releases take the date of the commit.